<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gest_billet</class>
 <widget class="QDialog" name="gest_billet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1315</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1301</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">#page{
	background-color: rgb(255, 255, 255);
	background-image: url(:/new/prefix1/lh.jpg);

border : 3px solid gray;
border-radius : 40px;
}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>331</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
border : 3px solid rgb(7, 202, 255);
border-radius : 40px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Num voyage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 19pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Formulaire</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modify">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 244, 0);
background-color: rgb(170, 255, 127);
QPushButton
{
background-color:rgb(85,170,255);
	color: rgb(0, 0, 0);
border-style: solid;
    border-radius: 6;
}
QPushButton:hover
{
background-color:rgb(85,110,255);
	color: rgb(0, 0, 0);
}
QPushButton:pressed
{
background-color:rgb(85,170,255);
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Emplacement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emp">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>num vouyageur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="voy">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>premiere classe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>classe affaires</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>classe eco</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>290</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>440</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 151, 6);
font: 63 23pt &quot;Yu Gothic UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>gestion billet</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>861</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QWidget{background:transparent} QTabWidget::pane{border: 1px;border-color:redt;background-color: transparent;} 
QTabBar::tab {background-color: transparent;color: #ccc;width: 100px;height:20px;font-size:24px} 
QTabBar::tab:hover{background-color:  rgb(255, 151, 6); color: white;}
QTabBar::tab:selected{background-color:  rgb(7, 202, 255); color: white;}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <property name="styleSheet">
       <string notr="true">#tab1{
border : 3px solid rgb(7, 202, 255);
border-radius : 40px;
}</string>
      </property>
      <attribute name="title">
       <string>menu</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>40</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>420</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Generer QR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supp">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>310</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>841</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>310</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="pixmap">
        <pixmap>:/shit/6424.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#comboBox_3{
border : 3px solid rgb(7, 202, 255);
border-radius : 40px;}</string>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_VOY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_VOL</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TRIER PAR</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="imp_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>350</y>
         <width>361</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 170, 255);</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>340</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_2
{border : 3px solid rgb(7, 202, 255);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="qrcode">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>380</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>Generer Mail</string>
       </property>
      </widget>
      <widget class="QPushButton" name="arduino">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>340</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>arduino</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab2">
      <property name="styleSheet">
       <string notr="true">#tab2{
border : 3px solid rgb(7, 202, 255);
border-radius : 40px;
}</string>
      </property>
      <attribute name="title">
       <string>mail</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>destinataire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>objet</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>votre message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>411</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>400</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>envoyer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>170</y>
         <width>411</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>170</y>
         <width>411</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="file">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browseBtn">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>140</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 127);</string>
       </property>
       <property name="text">
        <string>browse..</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 151, 6);
font: 63 13pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>fichier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true">#tab{
border : 3px solid rgb(7, 202, 255);
border-radius : 40px;
}</string>
      </property>
      <attribute name="title">
       <string>stat</string>
      </attribute>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>601</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(98, 213, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>400</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>premiere classe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>classe affaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>370</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>classe eco</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
